Gems/ROS2/Code/Source/Lidar/LidarSensorConfiguration.h

```diff
@@ -39,13 +40,20 @@ namespace ROS2
         AZStd::unordered_set<AZ::u32> m_ignoredCollisionLayers;
         AZStd::vector<AZ::EntityId> m_excludedEntities;
 
+        AZStd::vector<AZStd::tuple<AZStd::string, uint8_t, AZ::Color> > m_segmentationClasses = {
```

Comment:
Rather as a TODO for later, but you can probably handle new tags being created, and update m_segmentationClasses automatically.

Reply:
Seems like my misunderstanding or something related to updating color lookup at UI event - I am sorry for the confusion.